File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/php/valid.php
<?php
// Fixture — café 日本語 before declarations
namespace Fixture;

use InvalidArgumentException;

/** A greeter. */
class Greeter
{
    /** Say hello. */
    public function hello(string $name): string { return "hi $name"; }
}

interface Shape { public function area(): float; }

trait Loggable { public function log(string $m): void {} }

enum Color { case Red; case Blue; }

function helper(int $x): int { return $x + 1; }

File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/dart/valid.dart
// Fixture — café 日本語 before declarations
import 'dart:math';

/// A greeter.
class Greeter {
  /// Say hello.
  String hello(String name) => 'hi $name';
  Greeter();
}

abstract class Shape {
  double area();
}

mixin Loggable {
  void log(String m) {}
}

enum Color { red, blue }

typedef Callback = void Function(int);

int helper(int x) => x + 1;
